theseus/labs/lie/__init__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+from .types import ltype, SE3, SO3
+from .lie_tensor import (  # usort: skip
+    LieTensor,
+    adj,
+    compose,
+    exp,
+    hat,
+    inv,
+    jcompose,
+    jexp,
+    jinv,
+    jlog,
```

Comment:
do we want to add the other jacobinas (jadjoint, etc) even if they all point to _no_jop at the moment?

Reply:
Now that I think about it, the question is if we plan to have support for this in the future. If some of these operations don't make sense, I think it's better to completely remove them. We can then add the remaining ones both to the class and global scopes.

I think @fantaosha can make the choice of which ones.

Reply:
Makes sense to differentiate ones that can/will be added vs ones that don't work theoretically. It might be a good idea to still keep around all of them and raise a different error depending on not implemented or not possible.

Reply:
Yeah on second thought, you are right, it doesn't seem useful.

Reply:
> Now that I think about it, the question is if we plan to have support for this in the future. If some of these operations don't make sense, I think it's better to completely remove them. We can then add the remaining ones both to the class and global scopes.
> 
> I think @fantaosha can make the choice of which ones.

It depends. Some of these operators do not make sense at all. But some like `jadjoint` does have geometric meaning. We need to go back to this later.

Reply:
For now, I'll just keep the ones we have.